src/nidcpower/metadata/attributes.py

```diff
@@ -102,7 +102,7 @@
     },
     1050511: {
         'access': 'read only',
-        'channel_based': False,
+        'channel_based': True,
```

Comment:
It's possible these changes to `channel_based` key were made in internal metadata for the CXP feature. If that's the case, these changes should happen when the CXP support is added in nimi-python.

Reply:
I noticed that we some existing tech-debt in this area (TechinicalDebt# 1131303). I have manually reverted these changes.